docs/docs/adding-comments.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Adding comments
+---
+
+If you're using Gatsby to run a blog and you've started adding some content to it, the next thing to think about is how to increase engagement among your visitors. A great way to do that is to allow them to ask questions and express their views on what you've written. This will make your blog seem much more lively to anyone visiting it.
+
+There are many options out there for adding comment functionality to a site, several of them specfically targeted at static sites. So the hardest part may be to choose which one to go with. With no claims on completeness, here's a list of available services:
+
+- [Disqus](https://disqus.com)
+- [Facebook comments](https://www.npmjs.com/package/react-facebook)
+- [Staticman](https://staticman.net)
+- [JustComments](https://just-comments.com)
+- [TalkYard](https://www.talkyard.io)
+- [Gitalk](https://gitalk.github.io)
+
+All of the above are excellent options worth checking out. Staticman, for instance, took an interesting approach. Essentially, you set up your own HTML form for writing comments, let it send a POST request on submission to one of their endpoints. From this Staticman will automatically submit a pull request to your site's repo which you can accept or deny. This has the big advantage of keeping everything static (hence the name). All your data is in one place (your repo) as opposed to having to be loaded through JavaScript embeds or iframes on the fly. It will remain there even if Staticman is ever discontinued. With most other services, you depend on an external platform to deliver your comments. Of course, in return you have the increased manual setup of putting together the comment form and hooking it up to Staticman. Depending on your use case, this degree of customizability might be an advantage.
+
+But in this guide, we'll assume you want something fast with as little manual configuration and setup as possible. Disqus has several things going for it in this regard:
```

Comment:
Kind of a strange pivot here (e.g. we start with how cool Staticman is, and then immediately demonstrate Disqus). Let's maybe just lead with Disqus and then mention "other solutions" as interesting/novel/etc. at the end?

Reply:
You're right and thanks for pointing this out. I moved that paragraph into a comment at the bottom as a starting point for anyone who later intends to expand this guide to include other services [as suggested](https://github.com/gatsbyjs/gatsby/issues/12209#issuecomment-469724289) by @m-allanson.